docs/guides/src/main/server/configuration_overview.adoc

```diff
@@ -0,0 +1,177 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Configuring Keycloak"
+summary="An overview of the server configuration">
+
+In this guide, you will learn the core concepts around the server configuration and how to configure the server using the different
+server options available. You will also learn how to properly configure the server to achieve an optimal runtime for faster
+startup and low memory footprint.
+
+== Understanding Server Options
+
+All the server options can be found by using the `help` option for individual commands:
```

Comment:
```suggestion
All server options can be found by using the `help` option for individual commands:
```

Reply:
I changed that based on Andrew comment already.

Reply:
not hard on this, and works for me when @andymunro says it's better :)